index.html

```diff
@@ -310,6 +310,12 @@ <h3>Degradation preference when encoding</h3>
         encoding at the configured framerate and resolution, the encoder
```

Comment:
Suggested text:

When encoding video, the encoder is configured with a number of parameters; in this text, we'll call out resolution, framerate and "encoding parameters"; the latter is {{implementation-defined}}, but can influence both the quality of the resulting video, the resources required to encode, and the bitrate that the video consumes. Here, we talk about it as if a higher value gives better quality, but higher bitrate.
Normally, the UA will try to maximize all of those to give an optimal user experience.

When some constraint (bandwidth, CPU) prevents encoding with the very best parameters, the encoder....